pom.xml

```diff
@@ -17,25 +17,37 @@
     <packaging>pom</packaging>
 
     <properties>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <springboot.version>2.5.7</springboot.version>
-        <resteasy.version>5.0.0.Final</resteasy.version>
-        <resteasy.spring.version>2.0.0.Final</resteasy.spring.version>
-        <resteasy.jackson.version>2.12.3</resteasy.jackson.version>
+        <java.version>17</java.version>
         <jboss-logging.ver>3.4.0.Final</jboss-logging.ver>
-        <ver.mockito>3.12.4</ver.mockito>
-        <ver.powermock>2.0.9</ver.powermock>
-        <ver.objenesis>3.2</ver.objenesis>
-        <version.spring>5.3.13</version.spring>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
         <modular.jdk.args/>
         <modular.jdk.props/>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <resteasy.jackson.version>2.12.3</resteasy.jackson.version>
+        <resteasy.spring.version>3.0.0.Alpha1</resteasy.spring.version>
+        <resteasy.version>6.0.0.Final</resteasy.version>
+        <springboot.version>3.0.0-M1</springboot.version>
+        <ver.mockito>4.3.1</ver.mockito>
+        <ver.objenesis>3.2</ver.objenesis>
+        <!--        <ver.powermock>2.0.9</ver.powermock>-->
+        <version.jakarta.annotation.annotation-api>2.1.0-B1</version.jakarta.annotation.annotation-api>
+        <version.jakarta.servlet.servlet-api>5.0.0</version.jakarta.servlet.servlet-api>
+        <version.jakarta.ws.rs>3.0.0</version.jakarta.ws.rs>
+        <version.org.springframework>6.0.0-M2</version.org.springframework>
+        <ver.jakarta.validation-api>3.0.1</ver.jakarta.validation-api>
+        <version.org.eclipse>2.0.4</version.org.eclipse>
+        <ver.rest-assured>3.1.0</ver.rest-assured>
+        <ver.testng>6.14.3</ver.testng>
+        <ver.jakarta.xml.bind-api>4.0.0-RC3</ver.jakarta.xml.bind-api>
     </properties>
 
     <modules>
         <module>resteasy-spring-boot-starter</module>
         <module>sample-app</module>
-        <module>sample-app-for-wildfly</module>
+        <!--        <module>sample-app-for-wildfly</module>-->
```

Comment:
Does this not work?

Reply:
I haven't tried, and I'll give try to it.

@jamezp does wildfly support `jakarta.*` namespaces now?

Reply:
WildFly Preview does.